drivers/windows/inst/templates/task_run.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+# automatically generated
+echo generated on host: {{hostname}}
+echo generated on date: {{date}}
+echo hipercow version: {{hipercow_version}}
+echo hipercow.windows version: {{hipercow_windows_version}}
+echo running on: $(hostname -f)
+
+export PATH=/opt/apps/lmod/lmod/libexec:$PATH
+source /opt/apps/lmod/lmod/init/bash
+export LMOD_CMD=/opt/apps/lmod/lmod/libexec/lmod
+module use /modules-share/modules/all
+
+module load R/{{r_version}}
+module load hiredis
+module load libsodium
```

Comment:
what ends up needing libsodium?

Reply:
Oh - that's a mistake - it's hipercow.windows itself that needs it (keyring), but that's not needed on the node...